browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -511,7 +511,7 @@ class MarkdownNoteDetail extends React.Component {
           exportAsTxt={this.exportAsTxt}
           exportAsHtml={this.exportAsHtml}
           exportAsPdf={this.exportAsPdf}
-          wordCount={note.content.split(' ').length}
+          wordCount={note.content.replace(/\r?\n?\s+/g, ' ').trim().split(' ').length}
```

Comment:
Hello ! :)

I think it can be replaced by:

`note.content.trim().split(/\r?\n?\s+/g).length`

or even

`note.content.trim().split(/\s+/g).length`
since `\s` includes cariage returns and new lines

https://www.w3schools.com/jsref/jsref_regexp_whitespace.asp

Reply:
@thamara Can you change your code to the above suggestion for more concise code?  Thanks.

Reply:
Good point. I'd use the `split(/s+/g)`.

https://repl.it/@alexanderwolf/Word-count-regex

Reply:
I'll make the suggested change. Thanks!